tuneme/templates/_base.html

```diff
@@ -26,41 +26,50 @@
         {% endblock %}
 
         <script type="text/javascript">
-          WebFontConfig = {
-            google: { families: [ 'Exo+2:400,700:latin' ] }
-          };
-          (function() {
-            var wf = document.createElement('script');
-            wf.src = ('https:' == document.location.protocol ? 'https' : 'http') +
-              '://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js';
-            wf.type = 'text/javascript';
-            wf.async = 'true';
-            var s = document.getElementsByTagName('script')[0];
-            s.parentNode.insertBefore(wf, s);
-          })();
-
-          /*! loadJS: load a JS file asynchronously. [c]2014 @scottjehl, Filament Group, Inc. (Based on http://goo.gl/REQGQ by Paul Irish). Licensed MIT */
-        function loadJS( src, cb ){
-          "use strict";
-          var ref = window.document.getElementsByTagName( "script" )[ 0 ];
-          var script = window.document.createElement( "script" );
-          script.src = src;
-          script.async = true;
-          ref.parentNode.insertBefore( script, ref );
-          if (cb && typeof(cb) === "function") {
-            script.onload = cb;
+        //<![CDATA[
+        var menuItems = new Array();
+        function init() {
+          // Grab the accordion items from the page
+          var divs = document.getElementsByTagName( 'div' );
+          for ( var i = 0; i < divs.length; i++ ) {
+            // console.log(divs[i].className);
+            if ( divs[i].className == 'menuItem' ) menuItems.push( divs[i] );
+          }
+          // Assign onclick events to the accordion item headings
+          // console.log(menuItems)
+          for ( var i = 0; i < menuItems.length; i++ ) {
+            var span = getFirstChildWithTagName( menuItems[i], 'SPAN' );
+            // console.log("hello");
+            span.onclick = toggleItem;
+          }
+
+    	  // Hide all accordion item bodies except the first
+          for ( var i = 0; i < menuItems.length; i++ ) {
+            menuItems[i].className = 'menuItem hide';
           }
-          return script;
         }
+        function toggleItem() {
+          var itemClass = this.parentNode.className;
+
+    	  // Hide all items
+          for ( var i = 0; i < menuItems.length; i++ ) {
+            menuItems[i].className = 'menuItem hide';
+          }
 
-        // cut the mustard
-        if('querySelector' in document && 'localStorage' in window && 'addEventListener' in window) {
-          loadJS( "{% static 'js/main.js' %}" );
+    	  // Show this item if it was previously hidden
+          if ( itemClass == 'menuItem hide' ) {
+            this.parentNode.className = 'menuItem';
+          }
+        }
+        function getFirstChildWithTagName( element, tagName ) {
+          for ( var i = 0; i < element.childNodes.length; i++ ) {
+            if ( element.childNodes[i].nodeName == tagName ) return element.childNodes[i];
+          }
```

Comment:
@smn I’ve copied the javascript from the github page I’ve checked with milton’s work in the bwise he also removed them all so I assumed we don’t need them.